app/src/androidTest/java/org/dhis2/usescases/teidashboard/robot/EnrollmentRobot.kt

```diff
@@ -39,8 +40,9 @@ class EnrollmentRobot : BaseRobot() {
 
     fun clickOnPersonAttributes(attribute: String) {
         onView(withId(R.id.fieldRecycler))
-            .perform(actionOnItem<EditTextCustomHolder>(
-                hasDescendant(withText(attribute)), click()))
+            .perform(
+                scrollToBottomRecyclerView(),
+                actionOnItemAtPosition<RecyclerView.ViewHolder>(5, click()))
```

Comment:
why fixed position 5? This test might actually fail depending on the device screen.